manifests/05_role.yaml

```diff
@@ -106,53 +40,16 @@ metadata:
     capability.openshift.io/name: "marketplace"
 rules:
 - apiGroups:
-  - ""
-  resources:
-  - secrets
-  verbs:
-  - get
-- apiGroups:
-  - ""
-  resources:
-  - services
-  - serviceaccounts
-  verbs:
-  - get
-  - create
-  - delete
-  - update
-- apiGroups:
-  - ""
-  resources:
-  - configmaps
-  verbs:
-  - get
-  - create
-- apiGroups:
-  - ""
-  resources:
-  - pods
-  verbs:
-  - get
-- apiGroups:
-  - apps
-  resources:
-  - deployments
-  verbs:
-  - get
-  - create
-  - delete
-  - update
-- apiGroups:
-  - rbac.authorization.k8s.io
+  - operators.coreos.com
   resources:
-  - roles
-  - rolebindings
+  - catalogsources
   verbs:
   - get
   - create
   - delete
   - update
+  - list
+  - watch
```

Comment:
git diff is making it look like we're adding these verbs new here. Why do we need to add these verbs, if they're in fact new?

Reply:
Changing from RBAC to catalogsources in the rule